scripts/copyFilesUtils.mjs

```diff
@@ -91,11 +93,50 @@ export async function typescriptCopy({ from, to }) {
   return Promise.all(cmds);
 }
 
+export async function cjsCopy({ from, to }) {
+  if (!(await fse.pathExists(to))) {
+    console.warn(`path ${to} does not exists`);
+    return [];
+  }
+
+  const files = await glob('**/*.cjs', { cwd: from });
+  const cmds = files.map((file) => fse.copy(path.resolve(from, file), path.resolve(to, file)));
+  return Promise.all(cmds);
+}
+
 export async function createPackageFile() {
   const packageData = await fse.readFile(path.resolve(packagePath, './package.json'), 'utf8');
   const { nyc, scripts, devDependencies, workspaces, ...packageDataOther } =
     JSON.parse(packageData);
 
+  const modernCondition = 'mui-modern';
+  const legacyModernPrefix = './modern';
+
+  const packageExports = {
+    '.': {
+      types: './index.d.ts',
+      import: './index.mjs',
+      [modernCondition]: './index.modern.mjs',
+      require: './index.js',
+    },
+    './*': {
+      types: './*/index.d.ts',
+      import: './*/index.mjs',
+      [modernCondition]: './*/index.modern.mjs',
+      require: './*/index.js',
+    },
+    ...packageDataOther.exports,
+  };
+
+  const exportedNames = new Set(Object.keys(packageExports));
+  for (const exportedName of exportedNames) {
+    const modernName = exportedName.replace(/^\./, legacyModernPrefix);
```

Comment:
Maintain backwards compatibility for the legacy `./modern` bundles. Just a re-export of the main exports, but with a `./modern` prefix